src/pages/products/Products.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import './Products.css';
+
+
+class Products extends React  {
+
+ render() {
+    return (
+        <div>
+          
+            <h1 className="topic">Products</h1>
+        <div className="container d-flex justify-content-center mt-50 mb-50">
+        <div className="row">
+            <div className="col-md-4 mt-2">
+                <div className="card">
+                    <div className="card-body"> 
+                        <div className="card-img-actions"> <img src="" alt="Generic placeholder image"   className="card-img img-fluid" width="96" height="350"></img></div>
+                    </div>
+                    <div className="card-body bg-light text-center">
+                        <div className="mb-2">
+                            <h6 className="font-weight-semibold mb-2"> <a href="#" className="text-default mb-2" data-abc="true">Nike Running Wildhorse 7 Trail trainers in yellow</a> </h6> <a href="#" className="text-muted" data-abc="true">Free Delivery (Ts&Cs apply)</a>
+                        </div>
+                        <h3 className="mb-0 font-weight-semibold">$250.99</h3>
+                        <div> <i className="fa fa-star star"></i> <i className="fa fa-star star"></i> <i className="fa fa-star star"></i> <i className="fa fa-star star"></i> </div>
+                        <div className="text-muted mb-3">34 reviews</div> <button type="button" className="btn bg-cart"><i className="fa fa-cart-plus mr-2"></i> Add to cart</button>
+                    </div>
+                </div>
+            </div>
+            <div className="col-md-4 mt-2">
+                <div className="card">
+                    <div className="card-body">
+                        <div className="card-img-actions"><img src="" alt="Generic placeholder image"  className="card-img img-fluid" width="96" height="350"></img>
+                        </div>
+                    </div>
+                    <div className="card-body bg-light text-center">
+                        <div className="mb-2">
+                            <h6 className="font-weight-semibold mb-2"> <a href="#" className="text-default mb-2" data-abc="true">Nike Air Max 2090 trainers in luminus</a> </h6> <a href="#" className="text-muted" data-abc="true">Free Delivery (Ts&Cs apply)</a>
+                        </div>
+                        <h3 className="mb-0 font-weight-semibold">$50.99</h3>
+                        <div> <i className="fa fa-star star"></i> <i className="fa fa-star star"></i> <i className="fa fa-star star"></i> <i className="fa fa-star star"></i> </div>
+                        <div className="text-muted mb-3">34 reviews</div> <button type="button" className="btn bg-cart"><i className="fa fa-cart-plus mr-2"></i> Add to cart</button>
+                        </div>
+                     </div>
+            </div>
+            <div className="col-md-4 mt-2">
```

Comment:
No need to hard code multiple values. 1 hard coded one is also enough because we connect the UI with the API